### PR TITLE
update create event configuration

### PR DESCRIPTION
**Pull request by allisonnicoledeal**
Fixes https://github.com/cbsinteractive/propeller/issues/1527

### CODE REVIEW

File: elemental/templates/qvbr_mediastore.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<live_event href="/live_events/3" product="Elemental Live + HEVC Package" version="2.14.4.308149">
-  <name>{{channel_name}}</name>
+<live_event product="Elemental Live + HEVC Package" version="2.18.5.3019758">
```
Comment (allisonnicoledeal) [inline elemental/templates/qvbr_mediastore.xml:2]: Updated config to reflect August UEFA games, is this still what we want?

Comment (allisonnicoledeal) [reply elemental/templates/qvbr_mediastore.xml:2]: `<?xml version="1.0" encoding="UTF-8"?>
<live_event href="/live_events/2" product="Elemental Live" version="2.18.6.3020314">
  <name>Flavio/Toner test Oct 14 ucl-grstma1-oct20-17-amz-b</name>
  <input>
    <deblock_selected>true</deblock_selected>
    <denoise_selected>false</denoise_selected>
    <error_clear_time nil="true"/>
    <failback_rule>immediately</failback_rule>
    <filter_enable>Auto</filter_enable>
    <filter_strength>1</filter_strength>
    <hot_backup_pair>false</hot_backup_pair>
    <input_label></input_label>
    <loop_source>false</loop_source>
    <no_psi>false</no_psi>
    <order>1</order>
    <prefer_smpte2038>false</prefer_smpte2038>
    <service_name nil="true"/>
    <service_provider_name nil="true"/>
    <timecode_source>systemclock</timecode_source>
    <device_input>
      <device_type>AJA</device_type>
      <device_number>0</device_number>
      <channel>4</channel>
      <channel_type>HD-SDI</channel_type>
      <device_name>HD-SDI 4</device_name>
      <name nil="true"/>
      <sdi_settings>
        <input_format>Auto</input_format>
        <scte104_offset>0</scte104_offset>
      </sdi_settings>
    </device_input>
    <name>input_1</name>
    <video_selector>
      <color_space>follow</color_space>
      <default_afd nil="true"/>
      <order>1</order>
      <name>input_1_video_selector_0</name>
    </video_selector>
    <audio_selector>
      <default_selection>true</default_selection>
      <order>1</order>
      <program_selection>1</program_selection>
      <selector_type></selector_type>
      <unwrap_smpte337>true</unwrap_smpte337>
      <name>input_1_audio_selector_0</name>
    </audio_selector>
    <caption_selector>
      <order>1</order>
      <source_type>Embedded</source_type>
      <embedded_source_settings>
        <autodetect_scte20>true</autodetect_scte20>
        <source_608_channel_number>1</source_608_channel_number>
        <source_608_track_number>1</source_608_track_number>
        <upconvert_608_to_708>true</upconvert_608_to_708>
      </embedded_source_settings>
    </caption_selector>
  </input>
  <loop_all_inputs>false</loop_all_inputs>
  <timecode_config>
    <require_initial_timecode>false</require_initial_timecode>
    <source>systemclock</source>
    <sync_threshold nil="true"/>
  </timecode_config>
  <node_id>1</node_id>
  <failure_rule>
    <priority>50</priority>
    <restart_on_failure>false</restart_on_failure>
  </failure_rule>
  <input_loss_behavior>
    <black_frame_msec>10000</black_frame_msec>
    <input_loss_image_color>000000</input_loss_image_color>
    <input_loss_image_type>color</input_loss_image_type>
    <repeat_frame_msec>1000</repeat_frame_msec>
  </input_loss_behavior>
  <ad_trigger>scte35_splice_insert</ad_trigger>
  <ad_avail_offset>0</ad_avail_offset>
  <ignore_web_delivery_allowed_flag>false</ignore_web_delivery_allowed_flag>
  <ignore_no_regional_blackout_flag>false</ignore_no_regional_blackout_flag>
  <low_latency_mode>false</low_latency_mode>
  <initial_audio_gain>0</initial_audio_gain>
  <avsync_enable>true</avsync_enable>
  <avsync_pad_trim_audio>true</avsync_pad_trim_audio>
  <user_data></user_data>
  <input_end_action>switch_input</input_end_action>
  <output_timing_source>input_clock</output_timing_source>
  <input_buffer_size>60</input_buffer_size>
  <resource_reservation>none</resource_reservation>
  <low_framerate_input>false</low_framerate_input>
  <extract_sdt>false</extract_sdt>
  <stream_assembly>
    <name>stream_assembly_0</name>
    <video_description>
      <afd_signaling>None</afd_signaling>
      <drop_frame_timecode>true</drop_frame_timecode>
      <fixed_afd nil="true"/>
      <force_cpu_encode>false</force_cpu_encode>
      <height>1080</height>
      <insert_color_metadata>true</insert_color_metadata>
      <respond_to_afd>None</respond_to_afd>
      <sharpness>50</sharpness>
      <stretch_to_output>false</stretch_to_output>
      <timecode_passthrough>true</timecode_passthrough>
      <vbi_passthrough>false</vbi_passthrough>
      <width>1920</width>
      <h264_settings>
        <adaptive_quantization>medium</adaptive_quantization>
        <bitrate>35000000</bitrate>
        <buf_fill_pct nil="true"/>
        <buf_size>70000000</buf_size>
        <cabac>true</cabac>
        <dynamic_sub_gop>false</dynamic_sub_gop>
        <flicker_aq>true</flicker_aq>
        <force_field_pictures>false</force_field_pictures>
        <framerate_denominator nil="true"/>
        <framerate_follow_source>true</framerate_follow_source>
        <framerate_numerator nil="true"/>
        <gop_b_reference>false</gop_b_reference>
        <gop_closed_cadence>1</gop_closed_cadence>
        <gop_markers>false</gop_markers>
        <gop_num_b_frames>2</gop_num_b_frames>
        <gop_size>90.0</gop_size>
        <gop_size_units>frames</gop_size_units>
        <interpolate_frc>false</interpolate_frc>
        <look_ahead_rate_control>medium</look_ahead_rate_control>
        <max_bitrate nil="true"/>
        <max_qp nil="true"/>
        <min_bitrate nil="true"/>
        <min_buf_occ>0</min_buf_occ>
        <min_i_interval>0</min_i_interval>
        <min_qp nil="true"/>
        <motion_vector_direct_mode>spatial</motion_vector_direct_mode>
        <num_ref_frames>1</num_ref_frames>
        <par_denominator nil="true"/>
        <par_follow_source>true</par_follow_source>
        <par_numerator nil="true"/>
        <passes>1</passes>
        <progressive_references>false</progressive_references>
        <qp nil="true"/>
        <quality_level nil="true"/>
        <repeat_pps>false</repeat_pps>
        <rp2027_syntax>false</rp2027_syntax>
        <scd_mode>on</scd_mode>
        <sei_timecode>false</sei_timecode>
        <slices>0</slices>
        <slow_pal>false</slow_pal>
        <softness nil="true"/>
        <spatial_aq>true</spatial_aq>
        <svq>0</svq>
        <telecine>None</telecine>
        <temporal_aq>true</temporal_aq>
        <profile>Main</profile>
        <rate_control_mode>CBR</rate_control_mode>
        <interlace_mode>progressive</interlace_mode>
      </h264_settings>
      <selected_gpu nil="true"/>
      <codec>h.264</codec>
      <video_preprocessors>
        <deinterlacer>
          <algorithm>interpolate</algorithm>
          <deinterlace_mode>Deinterlace</deinterlace_mode>
          <force>false</force>
        </deinterlacer>
      </video_preprocessors>
    </video_description>
    <audio_description>
      <audio_type>0</audio_type>
      <follow_input_audio_type>true</follow_input_audio_type>
      <follow_input_language_code>true</follow_input_language_code>
      <language_code nil="true"/>
      <nielsen_rtvod_watermark nil="true"/>
      <order>1</order>
      <stream_name nil="true"/>
      <timecode_passthrough>false</timecode_passthrough>
      <aac_settings>
        <ad_broadcaster_mix>false</ad_broadcaster_mix>
        <bitrate>384000</bitrate>
        <coding_mode>2_0</coding_mode>
        <latm_loas>false</latm_loas>
        <mpeg2>false</mpeg2>
        <sample_rate>48000</sample_rate>
        <profile>LC</profile>
        <rate_control_mode>CBR</rate_control_mode>
      </aac_settings>
      <codec>aac</codec>
      <audio_source_name>Audio Selector 1</audio_source_name>
    </audio_description>
    <caption_description>
      <captions_subtype></captions_subtype>
      <destination_type>Embedded</destination_type>
      <dvb_sub_to_sd>false</dvb_sub_to_sd>
      <id3_as_caption_content>false</id3_as_caption_content>
      <language_code>ENG</language_code>
      <language_description>ENGLISH</language_description>
      <order>1</order>
      <caption_source_name>Caption Selector 1</caption_source_name>
    </caption_description>
  </stream_assembly>
  <output_group>
    <custom_name>reliable_ts</custom_name>
    <name nil="true"/>
    <order>1</order>
    <reliable_ts_group_settings>
      <on_input_loss>drop_ts</on_input_loss>
      <timed_metadata_id3_frame>PRIV</timed_metadata_id3_frame>
      <timed_metadata_id3_period>10</timed_metadata_id3_period>
      <name>reliable_ts_group_settings</name>
    </reliable_ts_group_settings>
    <type>reliable_ts_group_settings</type>
    <output>
      <arib_captions_passthrough>false</arib_captions_passthrough>
      <description nil="true"/>
      <ebif_passthrough>false</ebif_passthrough>
      <extension>m2ts</extension>
      <insert_amf_metadata>false</insert_amf_metadata>
      <insert_private_metadata>false</insert_private_metadata>
      <insert_scte35_esam>false</insert_scte35_esam>
      <insert_timed_metadata>false</insert_timed_metadata>
      <klv_passthrough>false</klv_passthrough>
      <log_edit_points>false</log_edit_points>
      <name_modifier nil="true"/>
      <nielsen_id3_passthrough>false</nielsen_id3_passthrough>
      <order>1</order>
      <scte35_passthrough>false</scte35_passthrough>
      <smpte_2038>false</smpte_2038>
      <start_paused>false</start_paused>
      <timed_metadata_passthrough>false</timed_metadata_passthrough>
      <reliable_ts_settings>
        <buffer_msec>1000</buffer_msec>
        <max_ts_packet_count>7</max_ts_packet_count>
        <mpts_membership>none</mpts_membership>
        <reliable_ts_type>zixi</reliable_ts_type>
        <destination>
          <uri>zixi://100.24.150.162:2088</uri>
        </destination>
      </reliable_ts_settings>
      <m2ts_settings>
        <arib>false</arib>
        <audio_frames_per_pes>2</audio_frames_per_pes>
        <bitrate>0</bitrate>
        <cc_in_pmt>false</cc_in_pmt>
        <drop_absent_streams>false</drop_absent_streams>
        <dvb>false</dvb>
        <ebp_on_audio>true</ebp_on_audio>
        <es_rate>false</es_rate>
        <fixed_ebp_audio_interval>false</fixed_ebp_audio_interval>
        <iframe_only>false</iframe_only>
        <null_packet_bitrate nil="true"/>
        <pat_interval>100</pat_interval>
        <pcr_every_pes>true</pcr_every_pes>
        <pcr_period nil="true"/>
        <pmt_interval>100</pmt_interval>
        <program_num>1</program_num>
        <scte35_pullup>0</scte35_pullup>
        <segmentation_markers>none</segmentation_markers>
        <segmentation_style>maintain_cadence</segmentation_style>
        <segmentation_time nil="true"/>
        <transport_stream_id nil="true"/>
        <use_atsc_stream_type>false</use_atsc_stream_type>
        <use_buffer_model>true</use_buffer_model>
        <vbr>false</vbr>
        <zixi_encryption>NO_ENCRYPTION</zixi_encryption>
        <zixi_keyvalue></zixi_keyvalue>
        <zixi_latency>6000</zixi_latency>
        <zixi_streamid>maste8567</zixi_streamid>
        <dvb_sdt_settings>
          <output_sdt>sdt_follow</output_sdt>
          <rep_interval>500</rep_interval>
          <service_name></service_name>
          <service_provider_name></service_provider_name>
        </dvb_sdt_settings>
        <audio_pids>482-498</audio_pids>
        <dvb_sub_pids>460-479</dvb_sub_pids>
        <scte27_pids>450-459</scte27_pids>
        <dvb_teletext_pid>499</dvb_teletext_pid>
        <pmt_pid>480</pmt_pid>
        <video_pid>481</video_pid>
      </m2ts_settings>
      <stream_assembly_name>stream_assembly_0</stream_assembly_name>
      <container>m2ts</container>
    </output>
  </output_group>
</live_event>
`

**Comment by allisonnicoledeal** [comment]
Use: https://cbs.slack.com/archives/G011XPLJLVD/p1602695825000600

File: setup.py [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-setuptools.setup(
-    name='python-elemental',
-    author='CBS Interactive',
-    author_email='video-processing-team@cbsinteractive.com',
-    version='0.4',
-    include_patckage_data=True,
-    url='https://github.com/cbsinteractive/elemental.git',
-    license='MIT',
-    long_description=open('README.md').read(),
-    install_requires=requirements,
```
Comment (rafaelcaricio) [inline setup.py:15]: Those deps were impacting Propeller... I split the dev and required dependencies now in Poetry.

File: elemental/client.py [hunk 3]
```diff
@@ -78,92 +78,69 @@ def send_request(self, http_method, url, headers, body="", timeout=None):
...
         body = "<stop></stop>"
-
-        # Generate headers according to how users create ElementalLive class
         headers = self.generate_headers(url)
+        self.send_request(http_method="POST", url=url, headers=headers, body=body, timeout=timeout)
 
-        # Send request and do exception handling
-        self.send_request(http_method="POST", url=url,
-                          headers=headers, body=body, timeout=timeout)
+    def reset_event(self, event_id, timeout=None):
```
Comment (rafaelcaricio) [inline elemental/client.py:120]: New action, we will probably need this when event goes to `fail` state.

**Review by igorsobreira — APPROVED** [review]

**Pull request merged**